tests/test_router_handover.cpp

```diff
@@ -49,16 +49,16 @@ void test_with_handover ()
     char my_endpoint[MAX_SOCKET_STRING];
     void *router = test_context_socket (ZMQ_ROUTER);
 
+	// Enable the handover flag
```

Comment:
Wrong indentation, please format all source files with clang-format, or apply the patch from the travis-ci clang-format job

Reply:
Ok.